From 0aab5aa1d39f65b856a96af2a06ebcace8db9fd2 Mon Sep 17 00:00:00 2001
From: Devin Lin <espidev@gmail.com>
Date: Sun, 18 Jan 2026 22:39:16 -0500
Subject: [PATCH] lockscreen: Fix failed login attempt causing no more
 authentication requests

For some reason at some point the kscreenlocker API rejects all
authentication requests if the grace lock (time after failed attempt ->
authenticator.startAuthentication) is shorter than 3 seconds, see
desktop: https://invent.kde.org/plasma/plasma-desktop/-/blob/a71b5db277bc9e8534a7b6e9412718f5e481e962/desktoppackage/contents/lockscreen/LockScreenUi.qml#L181

Update the grace lock interval to match desktop.
---
 shell/contents/lockscreen/LockScreenState.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/contents/lockscreen/LockScreenState.qml b/shell/contents/lockscreen/LockScreenState.qml
index d3d443c25..5125c60f1 100644
--- a/shell/contents/lockscreen/LockScreenState.qml
+++ b/shell/contents/lockscreen/LockScreenState.qml
@@ -59,7 +59,7 @@ QtObject {
     }
 
     property var graceLockTimer: Timer {
-        interval: 1000
+        interval: 3000
         onTriggered: {
             root.waitingForAuth = false;
             root.password = "";
-- 
GitLab

